venues/swsa.semanticweb.org/ISWC/2018/DeSemWeb/python/create-review-invitations.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python create-invitations.py --conf MyConf.org/2017 --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create authorGroup (can see reviews, can't write a review)
+#           reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+import os
+from openreview import *
+from openreview import tools
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print "Connecting to "+client.baseurl
+
+CONFERENCE_ID = 'swsa.semanticweb.org/ISWC/2018/DeSemWeb'
+PROGRAM_CHAIRS = CONFERENCE_ID+'/ProgramChairs'
+
+# TODO get real info 5/15/18 11:59 pm Hawaii time = 5/16/18 9:59am GMT
+REVIEW_TIMESTAMP =  tools.timestamp_GMT(2018,6,16,10)
+
+review_params = {
+    'readers': ['everyone'],
+    'writers': [CONFERENCE_ID],
+    'signatures': [CONFERENCE_ID],
+    'process': os.path.join(os.path.dirname(__file__), '../process/officialReviewProcess.js'),
+    'duedate': REVIEW_TIMESTAMP
+}
+
+review_content = {
```

Comment:
I think this is available in the invitations.content module

Reply:
or actually, maybe it's slightly different?

Reply:
You've approved these changes.  Can I go ahead and merge them?